pkg/gui/gui_config.go

```diff
@@ -0,0 +1,15 @@
+package gui
+
+import gui_server "github.com/Kong/kuma/pkg/config/gui-server"
+
+type GuiConfig struct {
```

Comment:
If `GuiConfig` is part of REST API, let's move into `types/` sub-package (like we do API server, Boostrap server, etc)